src/Node/Expression/Variable/TemplateVariable.php

```diff
@@ -12,12 +12,19 @@
 namespace Twig\Node\Expression\Variable;
 
 use Twig\Compiler;
-use Twig\Node\Expression\NameExpression;
+use Twig\Node\Expression\TempNameExpression;
 
-final class TemplateVariable extends NameExpression
+/**
+ * @final
```

Comment:
Is this intended? The added `GlobalTemplateVariable` extends this class. This currently makes the Symfony tests fail:

> ```
> Remaining indirect deprecation notices (1)
> 
>   1x: The "Twig\Node\Expression\Variable\TemplateVariable" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Twig\Node\Expression\Variable\GlobalTemplateVariable".
>     1x in ProfilerControllerTest::testPanelActionWithLatestToken from Symfony\Bundle\WebProfilerBundle\Tests\Controller
> ```

Reply:
Fixed in #4408